tests/structure/test_compare.py

```diff
@@ -29,4 +32,82 @@ def test_rmsf(stack, as_coord):
     if as_coord:
         stack = stack.coord
     assert struc.rmsf(struc.average(stack), stack).tolist() \
-           == pytest.approx([21.21320344] * 5)
\ No newline at end of file
+           == pytest.approx([21.21320344] * 5)
+
+def load_stack_superimpose():
```

Comment:
I think a `pytest` [fixture](https://docs.pytest.org/en/6.2.x/fixture.html) would better fit the purpose here as for example used [in this module](https://github.com/biotite-dev/biotite/blob/427b6eb398493da0f8c7a5c71acc20b2ca134c2d/tests/structure/test_basepairs.py#L31-L35).